README.md

```diff
@@ -1,10 +1,10 @@
-# Tyk-Sync
+# Tyk Sync
 
 ![Unstable packages](https://github.com/TykTechnologies/tyk-sync/workflows/Unstable%20packages/badge.svg)
 
 ## What is it?
 
-Tyk-Sync is a command line tool and library to manage and synchronise a Tyk installation with your version control system (VCS).
+Tyk Sync is a command line tool and library to manage and synchronise a Tyk installation with your version control system (VCS).
 
 About `tyk-git` - that was project's originally name and recently we have decided to change it to `tyk-sync` since it syncs to files as well as git repos.
```

Comment:
Can we take the opportunity to improve this, please?

I suggest:
`Note: the project was originally called `tyk-git` however this was changed to `tyk-sync` as it evolved and can now synchronise to files not just git repos.`